src/write_gcm.c

```diff
@@ -134,6 +134,9 @@ static _Bool wg_some_error_occurred_g = 0;
 // CreateTimeSeries request.
 #define MAX_TIME_SERIES_PER_REQUEST 200
 
+#define WG_CURL_FLAG_SILENT_FAILURES 1
```

Comment:
If these are bit flags that will be or'ed, it may be preferable to use `0x01` and `0x02`.